internal/controller/resources_cuup.go

```diff
@@ -33,7 +33,7 @@ type CuUpResources struct {
 
 func (resource CuUpResources) createNetworkAttachmentDefinitionNetworks(templateName string, ranDeploymentSpec *nephiov1alpha1.NFDeploymentSpec) (string, error) {
 	return free5gccontrollers.CreateNetworkAttachmentDefinitionNetworks(templateName, map[string][]nephiov1alpha1.InterfaceConfig{
-		"e1":  free5gccontrollers.GetInterfaceConfigs(ranDeploymentSpec.Interfaces, "e1-cu-up"),
+		"e1":  free5gccontrollers.GetInterfaceConfigs(ranDeploymentSpec.Interfaces, "e1"),
```

Comment:
Yes right. We need to propose these functions to a common place. Since now we have common NFDeployment, any functions defined for this, can be reused across all NFs,